### ISSUE TITLE
Decide on policy/guidelines for "Public Notifications"

### ISSUE THREAD

**Issue by stpaultim** [backdrop-ops/backdropcms.org#625]
We just added a content type for "Public Notifications" that will display on the Dashboard of any Backdrop CMS site with this block enabled (it will be enabled by default on any new installations with version 1.15 or later). 

We need to discuss and settle on some guidelines for what is appropriate content to be displaying here. I suggest we put this on agenda for next Outreach meeting. In the meantime, we can gather comments here. 

Currently, there are four Public Notifications published on BackdropCMS.org. The most recent 3 announcements are showing up on Backdrop sites. I think we should have a blog post about the new notifications system ready by January 15th and make sure that one of the notifications is a link to that post.

**Comment by stpaultim** [comment]
In addition to some editorial guidelines, I think we should assign an individual or small group of people who are responsible for curating content in this feed.

**Comment by jenlampton** [comment]
@stpaultim I think the list in the top post is perfect. :) I also like the idea for nominating a set of people who are responsible for creating these notices. 

Anyone else want to weigh in on the types of messages?

**Comment by BWPanda** [comment]
In my opinion/experience, I don't think these messages will really reach developers as much as the other audience listed. They might see them in passing while building a site, but I'm sure Backdrop developers already get their info elsewhere. So I think focus on the other 3 audiences.

As for content, I'm not sure about VUGs, but the others seem OK. But I'd suggest making sure they are only ever highly polished, set-in-stone announcements, posts, etc.

**Comment by klonos** [comment]
I also agree with @stpaultim's list, but I would also add announcements of PMC decisions.
